api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8770,4 +8770,205 @@
 		<!-- visit_attribute_type has not null constraint on uuid -->
 		<!-- visit_type has not null constraint on uuid -->
 	</changeSet>
+	
+	<changeSet id="20151006-1530" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set table
+		</comment>
+		<createTable tableName="order_set">
+			<column name="order_set_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="operator" type="varchar(50)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="name" type="varchar(255)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="description" type="varchar(1000)"/>
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
+			</column>
+			<column defaultValueBoolean="false" name="retired" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+			<column name="retired_by" type="int"/>
+			<column name="date_retired" type="DATETIME"/>
+			<column name="retire_reason" type="varchar(255)"/>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="DATETIME"/>
+			<column name="uuid" type="char(38)">
+				<constraints unique="true"/>
+			</column>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_set_creator"
+								 baseTableName="order_set" baseColumnNames="creator"
+								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_set_retired_by"
+								 baseTableName="order_set" baseColumnNames="retired_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_set_changed_by"
+								 baseTableName="order_set" baseColumnNames="changed_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+	</changeSet>
+	<changeSet id="20151006-1537" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set_member" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set_member table
+		</comment>
+		<createTable tableName="order_set_member">
+			<column name="order_set_member_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="order_type" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="order_template" type="text"/>
+			<column name="order_set_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="sort_weight" type="int">
```

Comment:
Does this _have_ to be an `int`? Can Hibernate work just as well with a `double`? [All other `sort_weight` attributes in the data model](http://resources.openmrs.org/datamodel/openmrs-data-model-1.11.html#.sort_weight) are `double` (well, `form_field.sort_weight` is a `real`, but it _should_ be a double but hasn't yet been fixed).